### DISCUSSION TITLE
Remove Client Data From Default Logs

### DISCUSSION THREAD

**Discussion by uslava77** [IvanJosipovic/BlazorApplicationInsights#186] [Q&A] [answered]
Hello,

I'm wondering if there is a way to remove the following fields from the default logs (pageView, dependencies, exceptions):
![image](https://user-images.githubusercontent.com/13425487/235761164-a62e88e5-374e-4c6c-bfed-7d925bf78993.png)

The goal is that none of the `client_` data fields are sent from client browsers to AI. 
This is for BlazorWASM.

Thanks!

**Comment by IvanJosipovic** [comment, marked as answer]
Hey,

See this link, https://github.com/microsoft/ApplicationInsights-JS/issues/1360
I would do this in the index.html, this way every request will be masked. If you do this in Blazor, it will still leak some through as Blazor executes after the JS is loaded.
